src/cli.c

```diff
@@ -402,6 +406,98 @@ static void get_info_callback(uv_work_t *work_req, int status)
     free(work_req);
 }
 
+static void set_auth()
+{
+    char *user = NULL;
+    char *user_input = NULL;
+    size_t user_input_size = 1024;
+    size_t num_chars;
+    user_input = calloc(user_input_size, sizeof(char));
+    if (user_input == NULL) {
+        printf("Unable to allocate buffer");
+        exit(1);
+    }
+    printf("Username (email): ");
+    num_chars = getline(&user_input, &user_input_size, stdin);
+    user = calloc(num_chars - 1, sizeof(char));
+    memcpy(user, user_input, num_chars * sizeof(char) - 1);
+
+    printf("Password: ");
+    char *pass = calloc(BUFSIZ, sizeof(char));
+    get_password(pass);
+    printf("\n");
+
+    char *mnemonic;
+    char *mnemonic_input;
+    size_t mnemonic_input_size = 1024;
+    mnemonic_input = calloc(mnemonic_input_size, sizeof(char));
+    if (mnemonic_input == NULL) {
+        printf("Unable to allocate buffer");
+        exit(1);
+    }
+    printf("Mnemonic: ");
+    num_chars = getline(&mnemonic_input, &mnemonic_input_size, stdin);
```

Comment:
`getline` isn't available for MinGW builds, there is now `get_input` available within the cli.